contrib/opencensus-ext-stackdriver/opencensus/ext/stackdriver/stats_exporter/__init__.py

```diff
@@ -42,6 +46,25 @@
 EPOCH_PATTERN = "%Y-%m-%dT%H:%M:%S.%fZ"
 GLOBAL_RESOURCE_TYPE = 'global'
 
+# OC metric descriptor type to SD metric kind and value type
+OC_MD_TO_SD_TYPE = {
```

Comment:
There should be one more mapping for `Summary` type. See Java: https://github.com/census-instrumentation/opencensus-java/pull/1591. You can file an issue to add support for it later.

Reply:
Opened https://github.com/census-instrumentation/opencensus-python/issues/567.